demos/JWST/S6_template.ecf

```diff
@@ -11,26 +13,25 @@ x_unit          um # Options include any measurement of light included in astrop
 # Scale height parameters (if you want a second copy of the plot with a second y-axis with units of scale height)
 star_Rad        0.6506 # The radius of the star in units of solar radii
 planet_Teq      1400 # The equilibrium temperature of the planet in units of Kelvin
-planet_Mass     2 # The planet's mass in units of Jupiter radii (used to calculate surface gravity)
+planet_Mass     2 # The planet's mass in units of Jupiter masses (used to calculate surface gravity)
 planet_Rad      None # The planet's radius in units of Jupiter radii (used to calculate surface gravity); Set to None to use the average fitted radius
 planet_mu       2.3 # The mean molecular mass of the atmosphere (in atomic mass units)
 planet_R0       None # The reference radius (in Jupiter radii) for the scale height measurement; Set to None to use the mean fitted radius
 
 # Diagnostics
 isplots_S6      5 # Generate few (1), some (3), or many (5) figures (Options: 1 - 5)
-testing_S6      False
 hide_plots      False # If True, plots will automatically be closed rather than popping up
 
 # Project directory
 topdir          /home/User/
 
-# Model to plot as well (path relative to project dir of a csv with col 1 as wavelength in microns, col 2 in the same units as the plotted spectrum, any headers preceded by a #)
-model_spectrum  None # Set to None if no model should be plotted
+# Model to plot underneath the fitted data points
```

Comment:
Normally, I plot the model **on top of** the data, but I can see instances where we might want to do plot the data underneath.  At some point, we should consider providing an option to modify the model's `zorder`.

Reply:
I was biased the same way, but the problem is that the models will tend to be much higher resolution than the data (at least in my case), and as a result the data gets hidden. I'll open a new issue to allow the data to be plotted on top of or below the model

Reply:
For example:

![fig6101_transmission](https://user-images.githubusercontent.com/24514122/168144027-28683c22-cb1d-4a5e-8b09-b6b4daf59d12.png)

Reply:
Agreed, it all depends on the resolution of the model.